### PR TITLE
Added in JavaScript Assignment 01 & 02

### PR DESCRIPTION
**Pull request by Ashwin2812**

### CODE REVIEW

File: Ashwin/JavaScript/02_Assignment/02_Assignment_1.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                border: 1px solid black;
+                background-image: url("Images/Minimalist-Crumpled-Paper-Simple-Background-Image.jpg");
+                margin-top: 150px;
+            }
+            button {
+                margin-top: 30px;
+            }
+        </style>
+    </head>
+
```
Comment (nileshp1122) [inline Ashwin/JavaScript/02_Assignment/02_Assignment_1.html]: remove extra line

File: Ashwin/JavaScript/02_Assignment/02_Assignment_2.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            function fun() {
+                first = parseInt(document.getElementById("input1").value);
+                second = parseInt(document.getElementById("input2").value);
+                third = parseInt(document.getElementById("input3").value);
+                fourth = parseInt(document.getElementById("input4").value);
+                fifth = parseInt(document.getElementById("input5").value);
+                function nextBiggest(arr) {
+                    var max = -Infinity,
+                        result = -Infinity;
+
```
Comment (nileshp1122) [inline Ashwin/JavaScript/02_Assignment/02_Assignment_2.html:21]: remove extra line

File: Ashwin/JavaScript/02_Assignment/02_Assignment_2.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                    for (const value of arr) {
+                        const nr = Number(value);
+
```
Comment (nileshp1122) [inline Ashwin/JavaScript/02_Assignment/02_Assignment_2.html:24]: remove extra line

File: Ashwin/JavaScript/02_Assignment/02_Assignment_2.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                        if (nr > max) {
+                            [result, max] = [max, nr];
+                        } else if (nr < max && nr > result) {
+                            result = nr;
+                        }
+                    }
+
```
Comment (nileshp1122) [inline Ashwin/JavaScript/02_Assignment/02_Assignment_2.html]: remove extra line

File: Ashwin/JavaScript/02_Assignment/02_Assignment_2.html [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+                    document.getElementById("h").innerHTML = result;
+                }
+
```
Comment (nileshp1122) [inline Ashwin/JavaScript/02_Assignment/02_Assignment_2.html]: remove extra line

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

File: Ashwin/JavaScript/02_Assignment/02_Assignment_2.html [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+            function fun() {
+                first = parseInt(document.getElementById("input1").value);
+                second = parseInt(document.getElementById("input2").value);
+                third = parseInt(document.getElementById("input3").value);
+                fourth = parseInt(document.getElementById("input4").value);
+                fifth = parseInt(document.getElementById("input5").value);
+                function nextBiggest(arr) {
+                    var max = -Infinity,
+                        result = -Infinity;
+
```
Comment (shubhamdalal49) [inline Ashwin/JavaScript/02_Assignment/02_Assignment_2.html:21]: remove this extra line

File: Ashwin/JavaScript/02_Assignment/02_Assignment_2.html [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+                    for (const value of arr) {
+                        const nr = Number(value);
+
```
Comment (shubhamdalal49) [inline Ashwin/JavaScript/02_Assignment/02_Assignment_2.html:24]: remove this extra line

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Pull request merged**